### PR TITLE
hey, if you type in your pw, it will show as stars

### PR DESCRIPTION
**Pull request by 81Denton**
only '80s kids will remember this

### CODE REVIEW

**Comment by Fikou** [comment]
boomer321

**Comment by Fikou** [comment]
bro wtf

File: code/modules/antagonists/traitor/equipment/Malf_Modules.dm [hunk 1]
```diff
@@ -299,7 +302,10 @@ GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
 	sleep(14)
 	if(!owner || QDELETED(owner))
 		return
-	to_chat(owner, "<span class='small boldannounce'>******************</span>")
+	if(prob(10))
+		to_chat(owner, "<span class='small boldannounce'>hunter2</span>")
```
Comment (AffectedArc07) [inline code/modules/antagonists/traitor/equipment/Malf_Modules.dm]: Can you not publicly post my pass thanks

Comment (81Denton) [reply code/modules/antagonists/traitor/equipment/Malf_Modules.dm]: All I see is *************

Comment (AffectedArc07) [reply code/modules/antagonists/traitor/equipment/Malf_Modules.dm]: Ah ok good

**Comment by Fikou** [comment]
you should make the stars the same length as the password

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/antagonists/traitor/equipment/Malf_Modules.dm [hunk 2]
```diff
@@ -266,7 +266,10 @@ GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
 	sleep(3)
 	if(!owner || QDELETED(owner))
 		return
-	to_chat(owner, "<span class='small boldannounce'>auth 'akjv9c88asdf12nb' ******************</span>")
+	if(prob(10))
```
Comment (optimumtact) [inline code/modules/antagonists/traitor/equipment/Malf_Modules.dm]:
```
var/pass = prob(10) ? "******" : "hunter2"
to_chat(owner, "<span class='small boldannounce'>auth 'akjv9c88asdf12nb' [pass]</span>")
to_chat(owner, "<span class='small boldannounce'>[pass]</span>")
```
?

**Comment by Shadowflame909** [comment]
who the heck is ~hunter2~ *******

**Comment by Anonmare** [comment]
WTF why are you posting my password on the git?


Also I can't tell if this is a reference to the original leak back in '03 or not.

**Comment by GunHog** [comment]
> WTF why are you posting my password on the git?
> 
> Also I can't tell if this is a reference to the original leak back in '03 or not.

I believe this is in reference to an old RuneScape social engineering technique where scammers would state that the chat system censors passwords. The idea was to get someone gullible enough to say his password, so the scammer could hijack that account.

File: code/modules/antagonists/traitor/equipment/Malf_Modules.dm (hunk 2 above)
Comment (81Denton) [reply code/modules/antagonists/traitor/equipment/Malf_Modules.dm]: good idea

**Comment by 81Denton** [comment]
I'm one of those heathens that never played Runescape, so I just know it from bash.org: http://bash.org/?244321

**Comment by Tharcoonvagh** [comment]
Why do dumb people use the internet

**Comment by Time-Green** [comment]
> Why do dumb people even use the internet

To post that comment

**Comment by optimumtact** [comment]
holy shit you fuckign zoomers, the bash.org reference came first

**Comment by cacogen** [comment]
urgh

**Review by Cyberboss — APPROVED** [review]

**Comment by MCterra10** [comment]
Lmao this is based

**Pull request merged**